### DISCUSSION TITLE
Retain section columns when combining docs with body_add_docx()

### DISCUSSION THREAD

**Discussion by akretchun** [davidgohel/officer#471] [Q&A] [answered]
Hi all - 

@davidgohel First, thanks so much for all the work/dev on this package, its helping us a ton with automated reporting. 

I had a question, somewhat related to [this issue](https://github.com/davidgohel/officer/issues/431) regarding section formatting when using multiple body_add_docx() calls. I'm wondering if there is a way to retain multiple columns when you combine multiple docx files that all have multiple columns. Currently, using read_docx() on the first docx retains multiple columns, but subsequent body_add_docx() calls in the same pipe will remove the column definitions of all added docxs.

Any help or guidance would be much appreciated!

```
library(officer) 

file1 <- "file1.docx"
file2 <- "file2.docx"
file3 <- "file3.docx"

###dummy text
txt_lorem <- rep(
  "Purus lectus eros metus turpis mattis platea praesent sed. ",
  200)
txt_lorem <- paste0(txt_lorem, collapse = "")

txt_quickfox <- rep(
  "The quick brown fox jumped overthe lazy dog. ",
  200)
txt_quickfox <- paste0(txt_quickfox, collapse = "")

###section properties
sec.columns <- section_columns(widths = c(3, 3), space = 0.25)

ps_section <- prop_section(
  type = "nextPage",
  section_columns = sec.columns
)

###create and combine docxs
x <- read_docx()
x <- x %>% 
  body_add_par(txt_lorem) %>%
  body_end_block_section(block_section(property = ps_section)) %>%
  print(x, target = file1)

x <- read_docx()
x <- x %>% 
  body_add_par(txt_quickfox) %>%
  body_end_block_section(block_section(property = ps_section)) %>%
  print(x, target = file2)

x <- read_docx(path = file1) %>%
  body_end_block_section(block_section(property = ps_section)) %>%
  body_add_docx(src = file2) %>%
  body_end_block_section(block_section(property = ps_section)) %>%
  print(x, target = file3)
```

**Comment by davidgohel** [comment, marked as answer]
yes, as you said, it's a duplicated of #431

> you have to set the sections [...] at merge time

> This is a Word feature/issue we can not fix
